<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Network_Interfaces">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Network Interfaces</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="textInterfaces"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>341</width>
      <height>247</height>
     </rect>
    </property>
    <property name="title">
     <string>Modify IP Address</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>IP Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editIP">
       <property name="placeholderText">
        <string>e.g., 192.168.1.100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Subnet Mask:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editSubnet">
       <property name="placeholderText">
        <string>e.g., 255.255.255.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Interface Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboInterface">
       <property name="placeholderText">
        <string>eth0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>951</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Activate/Deactivate Interface</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnActivate">
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeactivate">
       <property name="text">
        <string>Deactivate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Interface Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboActivateDeactivate">
       <property name="placeholderText">
        <string>eth0 </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
